paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -77,6 +79,7 @@
 DEFAULT_DEPLOYMENT_TIMEOUT = 3600  # seconds
 DEFAULT_AUTO_CERTIFY_DELAY = 600  # seconds
 DEFAULT_SLACK_CHANNEL = "#deploy"
+DEFAULT_ALLOWED_PUSH_GROUPS = os.environ.get("PAASTA_SECRET_ALLOWED_PUSH_GROUPS", None)
```

Comment:
should this also be in SystemPaastaConfig?

Reply:
+1. Fixed.